### PR TITLE
Add extra public fn to grab seq_num from RequestQueue

### PR DESCRIPTION
**Pull request by filipzeta**

### CODE REVIEW

File: dex/src/state.rs [hunk 1]
```diff
@@ -871,6 +871,10 @@ impl RequestQueue<'_> {
         self.header.next_seq_num += 1;
         seq_num
     }
+
+    pub fn get_next_seq_num(& self) -> u64 {
```
Comment (xjcaa) [inline dex/src/state.rs:875]: &self

Also why not just also add `generate_order_id`?

Comment (filipzeta) [reply dex/src/state.rs]: Thoughts on this change? I just made `gen_order_id` public and added a `increment_seq: bool`. It's only used once aside from the code I'm adding to DEX.

Comment (filipzeta) [reply dex/src/state.rs]: Thoughts on this change? I just made `gen_order_id` public and added a `increment_seq: bool`. It's only used once aside from the code I'm adding to DEX.

**Review by xjcaa — APPROVED** [review]
lgtm

**Pull request merged**